src/utils/hooks/useRestartReview.ts

```diff
@@ -25,23 +31,36 @@ const useRestartReview: UseRestartReview = ({ reviewId, structure, assignment })
 
   const constructReviewPatch: ConstructReviewPatch = (structure) => {
     const elements = Object.values(structure?.elementsById || {})
-
+    // Check for draft review response in case consolidation is ongoing and new review response is added
```

Comment:
Hmmm nice! This should probably part of [Epic #79 - Add new review to consolidation](https://github.com/openmsupply/application-manager-web-app/issues/537) already implemented here : D